proto/account.proto

```diff
@@ -3,30 +3,45 @@ package account;
 
 import "digest.proto";
 
+// An account ID.
 message AccountId {
     // A Miden account ID is a 120-bit value derived from the commitments to account code and
     // storage, and a random user-provided seed.
     bytes id = 1;
```

Comment:
Should this comment be for the entire message rather than for the individual field?